src/common/adjacency_ransac.h

```diff
@@ -39,10 +39,12 @@
 #include <map>
 #include <vector>
 
-#include <opencv2/core/core.hpp>
-
```

Comment:
The latest OpenCV3 should be fine with that file, you don't need to modify it. Thx